desktop/windows/release-notes/index.md

```diff
@@ -23,15 +23,27 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
-## Docker Desktop 4.7.0
-2022-04-07
+## Docker Desktop 4.7.1
+2022-04-19
 
 > Download Docker Desktop
 >
 > [For
 > Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
 > .button .primary-btn }
 
+### Bug fixes and minor changes
+ - Fixed a bug where update was failing with a symlink error. Fixes [docker/for-win#12650](https://github.com/docker/for-win/issues/12650).
+ - Fixed a bug that prevent starting in windows contianer mode because some layers contained symlinks. Fixes [docker/for-win#12652](https://github.com/docker/for-win/issues/12652).
```

Comment:
```suggestion
 - Fixed a bug that prevented starting Docker Desktop in Windows container mode because some layers contained symlinks. Fixes [docker/for-win#12652](https://github.com/docker/for-win/issues/12652).
```